ipyparallel/client/asyncresult.py

```diff
@@ -134,6 +134,27 @@ def _init_futures(self):
         self._output_future.add_done_callback(self._resolve_output)
         self.add_done_callback(self._finalize_result)
 
+    @staticmethod
+    def io_stream_callback(ar, eid, msg):
```

Comment:
I think this should be a regular instance method, especially if `self` (ar) is used

Reply:
Makes sense, done.